config/services.yml

```diff
@@ -27,7 +27,7 @@ https://cmr.earthdata.nasa.gov:
     description: |
       A service to compose the Giovanni URL and invoke Giovanni service to produce output file to visualize, analyze,
       and access vast amounts of Earth science remote sensing data without having to download the data.
-    data_operation_version: '0.17.0'
+    data_operation_version: '0.18.0'
```

Comment:
What do you all think about adding a variable at the top of the file for the latest schema version and then do the YAML equivalent of setting LATEST_SCHEMA_VERSION... that way we only update one place when we change the schema, and we make it more obvious if a service is pinned to an older version?

Reply:
Sounds good to me if there's a straightforward way to do it.